collaboration/sig-core/proposals/kyma-controller-manager.md

```diff
@@ -0,0 +1,181 @@
+# Consolidate Kyma controllers inside a single controller manager
+
+Created on 2019-10-11 by Antoine Cotten (@antoineco).
+
+## Status
+
+DRAFT. Proposed on 2019-10-11.
+
+## Motivation
+
+Kyma is composed of multiple components that share a lot of architectural patterns, yet are written in isolation
+following different engineering standards and using heterogeneous SDKs and libraries, before eventually being released
+as _one product_.
```

Comment:
I fully agree and that is also something which we should not only drive forward regarding the controllers. There are a lot of other more generic technical aspects (e.g. logging) which we should harmonize.